### ISSUE TITLE
How to append userIdentities at subsequent SignIns?

### ISSUE THREAD

**Issue by cyrilmatthew26** [azure-ad-b2c/samples#422]
Hi all,

Following the [auto account link sample](https://github.com/azure-ad-b2c/samples/tree/master/policies/auto-account-linking) I was able to successfully provision new social accounts with a local account

**My requirement is not to force the user to authenticate again with the existing local/social account and directly append the userIdentity after sign In attempt.** (Can we expect a separate sample for this soon?) Because of this particular requirement, I've removed the "xxxxx-OAUTH-Link" profiles and subsequent references in the "HandleLinkLocalToSocial" journey

However, when I attempt federated sign-up/sign-in for the second time using a different social provider the userIdentity is overriding the existing userIdentity, so in the end there's only one at any given time

**One of the problem I'm facing**

The currentUserIdentities outputClaim is being returned as undefined in the AAD-FindLocalAccountWithSocialEmail profile as per App Insight logs. So with that there's no way to add the incoming issuer id to the existing collection using "AppendNewUserIdentityToCurrentUserIdentities". Only the new issuer id is getting added.

Is there a way to append the 

Any suggestions?

**Comment by JasSuri** [comment]
In `HandleLinkLocalToSocial` subjourjey, remove steps 1-7 inclusive.

**Comment by cyrilmatthew26** [comment]
Thank you a lot for the prompt response @JasSuri . I got it working now.
I had made a few changes and renamed the flows previously. I had the steps correct, but then I started comparing the whole thing and found that I only missed <Item Key="api-version">1.6</Item> in the AAD-FindLocalAccountWithSocialEmail which was the problem.

Just curious, what difference does the api-version make in the backend?
